tests/resources/serializers/test_csl_serializer.py

```diff
@@ -39,3 +43,56 @@ def test_csl_serializer(
     serialized_record = serializer.dump_one(full_record)
 
     assert serialized_record == expected_data
+
```

Comment:
Would it be worth adding a tests for `serialize_object_list`? I see no other serializer test it. 🤔

Reply:
Good idea!